demo-android/src/main/java/nl/tudelft/ipv8/android/demo/coin/WalletManager.kt

```diff
@@ -45,14 +53,17 @@ class WalletManager(
         }
 
         val filePrefix = when (walletManagerConfiguration.network) {
-            BitcoinNetworkOptions.TEST_NET -> "forwarding-service-testnet"
-            BitcoinNetworkOptions.PRODUCTION -> "forwarding-service"
+            BitcoinNetworkOptions.TEST_NET -> TEST_NET_WALLET_NAME
+            BitcoinNetworkOptions.PRODUCTION -> MAIN_NET_WALLET_NAME
         }
 
         kit = object : WalletAppKit(params, walletDir, filePrefix) {
             override fun onSetupCompleted() {
                 // Make a fresh new key if no keys in stored wallet.
-                if (wallet().keyChainGroupSize < 1) wallet().importKey(ECKey())
+                if (wallet().keyChainGroupSize < 1) {
```

Comment:
This part is not needed anymore, we are using either seed or the new publicPrivateKeyPair  you added I think.